version.properties

```diff
@@ -1,2 +1,2 @@
-version=2.2.0-beta.2
+version=2.2.0
```

Comment:
I intend to push the button manually from BinTray Web interface once I smoke test the release. After few successful releases, I will put back automatic maven central.